tests/flow/requirements.txt

```diff
@@ -1,2 +1,5 @@
+# MacOS setup on github actions needs setuptools (required by RLTest).
+# Normally, it would be better RLTest have this package as dependency.
+setuptools
```

Comment:
Please move it to `./install/mariner2.sh` (`pip install --upgrade setuptools`) you can see what I did in `1.10`.
btw I think there are more issue with macos that we need to cp from `1.10`:
`2c418b0 2023-12-25 change mac-os-64 to 13.4.1 (YaacovHazan)`
`d2dfbc3 2023-12-30 fix macos python installation (YaacovHazan)`

but let's wait with it as I have a conversation about it with CTO...